pkgs/by-name/vo/voxinput/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  makeWrapper,
+  testers,
+  libpulseaudio,
+  dotool,
+  voxinput,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "voxinput";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "richiejp";
+    repo = "VoxInput";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-ykWb5I3cd3DMDVqYrcmOtCKhLpmob7HBXs5Ek5E7/do=";
+  };
+
+  vendorHash = "sha256-OserWlRhKyTvLrYSikNCjdDdTATIcWTfqJi9n4mHVLE=";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    libpulseaudio
+    dotool
+  ];
+
+  # To take advantage of the udev rule something like `services.udev.packages = [ nixpkgs.voxinput ]`
+  # needs to be added to your configuration.nix
+  postInstall = ''
+    mv $out/bin/VoxInput $out/bin/voxinput_tmp ; mv $out/bin/voxinput_tmp $out/bin/voxinput
```

```suggestion
    mv $out/bin/VoxInput $out/bin/voxinput_tmp ; mv $out/bin/voxinput_tmp $out/bin/voxinput
```

oof I am guessing it can't be directly moved

---

Better approach would be to have an symbolic link

so users migrating from other distros can use `VoxInput` just fine.

---

Not on case insensitive file systems (maybe wtih `-f` perhaps, which then could hide other issues) as per [Aleksanaa](https://github.com/Aleksanaa) review comment.